xray/commands/audit/scan.go

```diff
@@ -321,10 +327,10 @@ func collectPatternMatchingFiles(fileData spec.File, rootPath string, dataHandle
 	return nil
 }
 
-// Xray expect a path inside a repo, but not accpet path to a file.
+// Xray expect a path inside a repo, but not accept path to a file.
```

Comment:
Xray expect a path inside a repo, but not accept path to a file.
-->
Xray expects a path inside a repo, but does not accept a path to a file.